anitya/ui.py

```diff
@@ -430,6 +430,45 @@ def edit_project(project_id):
         current='projects',
         form=form,
         project=project,
+    )
+
+
+@APP.route('/project/<project_id>/flag', methods=['GET', 'POST'])
+@login_required
+def flag_project(project_id):
+
+    project = anitya.lib.model.Project.get(SESSION, project_id)
+    if not project:
+        flask.abort(404)
+
+    form = anitya.forms.FlagProjectForm(
+        obj=project)
+
+    if form.validate_on_submit():
+        try:
+            anitya.lib.flag_project(
+                SESSION,
+                project=project,
+                reason=form.reason.data,
+                user_mail=flask.g.auth.email,
+            )
+            flask.flash('Project flagged for admin review')
+            # TODO: what happens with justflag? Need to change something
+            # somewhere else for that to work?
+            flask.session['justflag'] = True
```

Comment:
`justflag`? Apparently, nothing. I swear I took that from somewhere else (I didn't just make it up), but it looks like it's gone now. Either way, it doesn't do anything apparently so I've deleted both instances of it. :+1: